tests/testsuite/package.rs

```diff
@@ -2538,18 +2516,14 @@ fn long_file_names() {
         .build();
 
     p.cargo("package").run();
-    p.cargo("package --list")
-        .with_stdout(&format!(
-            "\
-{}
+    p.cargo("package --list").with_stdout_data(str![[r#"
+012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789
```

Comment:
this is some magic number, could it be ignored?

not a block, just curious.

Reply:
Since this eliminates the need for `format!` and it can be automatically generated by `snapshot=overwrite`, I believe this approach is likely acceptable.

Please let me know if this is not appropriate. If so, I'll revert to using `format!` again.